.github/workflows/build-kfam.yml

```diff
@@ -0,0 +1,85 @@
+name: component/kfam
+on:
+  push:
+    branches:
+      - stc-master
+    paths:
+      - components/access-management/**
+  pull_request:
+    types:
+      - 'opened'
+      - 'reopened'
+# Environment variables available to all jobs and steps in this workflow
+env:
+  REGISTRY_NAME: k8scc01covidacr
+  DEV_REGISTRY_NAME: k8scc01covidacrdev
+  CLUSTER_NAME: k8s-cancentral-02-covid-aks
+  CLUSTER_RESOURCE_GROUP: k8s-cancentral-01-covid-aks
+
+jobs:
+  build-push:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    # Determine if pushing to Prod or Dev ACR
+    - name: Set ENV variables for a PR containing the auto-deploy tag
+      if: github.event_name == 'pull_request' && contains( github.event.pull_request.labels.*.name, 'auto-deploy')
+      run: echo "REGISTRY=${{env.DEV_REGISTRY_NAME}}.azurecr.io" >> "$GITHUB_ENV"
+   
+    - name: Set ENV variable for pushes to master
+      if: github.event_name == 'push' && github.ref == 'refs/heads/stc-master'
+      run: echo "REGISTRY=${{env.REGISTRY_NAME}}.azurecr.io" >> "$GITHUB_ENV"
+
+    # Connect to Azure Container registry (ACR)
+    - uses: azure/docker-login@v1
+      with:
+        login-server: ${{ env.REGISTRY_NAME }}.azurecr.io
+        username: ${{ secrets.REGISTRY_USERNAME }}
+        password: ${{ secrets.REGISTRY_PASSWORD }}
+        
+    # Connect to DEV Azure Container registry (ACR)
+    - uses: azure/docker-login@v1
+      with:
+        login-server: ${{ env.DEV_REGISTRY_NAME }}.azurecr.io
+        username: ${{ secrets.DEV_REGISTRY_USERNAME }}
+        password: ${{ secrets.DEV_REGISTRY_PASSWORD }}
+
+    - name: Free disk space
+      run: |
+        sudo swapoff -a
+        sudo rm -f /swapfile
+        sudo apt clean
+        docker rmi $(docker image ls -aq)
+        df -h
+
+    # Container build to a Azure Container registry (ACR)
+    - name: Docker build
+      run: |
+        docker build \
+          -t ${{ env.REGISTRY }}.azurecr.io/kubeflow/kfam:${{ github.sha }} \
```

Comment:
Using the {{env.REGISTRY}} here, you don't need to specifiy the `.azurecr.io` right after since that was already store in REGISTRY in the steps above. (see the build-centraldashboard.yml file).

Also, do we want to call this image "kfam" or "access-management"? I know kfam is a lot less characters to write, but this image is for the "components/access-management" part of the repo.

Reply:
Valid valid 👍